package com.xworkz.ExceptionalEvents.Runner;

import com.xworkz.ExceptionalEvents.classes.HundredEvents;

public class HundredEventsRunner {

	public static void main(String[] args) throws Exception {
		HundredEvents hundredEvents = new HundredEvents();
		
		hundredEvents.name();
		hundredEvents.gender();
		hundredEvents.Book();
		hundredEvents.place();
		hundredEvents.laptop();
		hundredEvents.computer();
		hundredEvents.bike();
		hundredEvents.chair();
		hundredEvents.charger();
		hundredEvents.phone();
		hundredEvents.powerBank();
		hundredEvents.power();
		hundredEvents.bank();
		hundredEvents.Corporation();
		hundredEvents.pen();
		hundredEvents.school();
		hundredEvents.college();
		hundredEvents.power();
		hundredEvents.bottle();
		hundredEvents.watch();
		hundredEvents.chair();
		hundredEvents.glass();
		hundredEvents.students();
		hundredEvents.fan();
		hundredEvents.door();
		hundredEvents.wire();
		hundredEvents.table();
		hundredEvents.metro();
		hundredEvents.train();
		hundredEvents.bus();
		hundredEvents.car();
		hundredEvents.bangalore();
		hundredEvents.mysore();
		hundredEvents.tv();
		hundredEvents.hp();
		hundredEvents.telephone();

		System.out.println("Running the error Exception");
		
		hundredEvents.Method1();
		hundredEvents.Method2();
		hundredEvents.Method3();
		hundredEvents.Method4();
		hundredEvents.Method5();
		hundredEvents.Method6();
		hundredEvents.Method7();
		hundredEvents.Method8();
		hundredEvents.Method9();
		hundredEvents.Method10();
		hundredEvents.Method11();
		hundredEvents.Method12();
		hundredEvents.Method13(); 
		hundredEvents.Method14();
		hundredEvents.Method15();
		hundredEvents.Method16();
		hundredEvents.Method17();
		hundredEvents.Method18();
		hundredEvents.Method19();
		hundredEvents.Method20();
		hundredEvents.Method21();
		hundredEvents.Method22();
		hundredEvents.Method23();
		hundredEvents.Method24();
		hundredEvents.Method25();
		hundredEvents.Method26();
		hundredEvents.Method27();
		hundredEvents.Method28();
		hundredEvents.Method29();
		hundredEvents.Method30();
		hundredEvents.Method31();
		hundredEvents.Method32();
		hundredEvents.Method33();
		hundredEvents.Method34();
		hundredEvents.Method35();
		
		System.out.println("Running the Exception error");
		
		hundredEvents.system1();
		hundredEvents.system2();
		hundredEvents.system3();
		hundredEvents.system4();
		hundredEvents.system5();
		hundredEvents.system6();
		hundredEvents.system7();
		hundredEvents.system8();
		hundredEvents.system9();
		hundredEvents.system10();
		hundredEvents.system11();
		hundredEvents.system12();
		hundredEvents.system13();
		hundredEvents.system14();
		hundredEvents.system15();
		hundredEvents.system16();
		hundredEvents.system17();
		hundredEvents.system18();
		hundredEvents.system19();
		hundredEvents.system20();
		hundredEvents.system21();
		hundredEvents.system22();
		hundredEvents.system23();
		hundredEvents.system24();
		hundredEvents.system25();
		hundredEvents.system26();
		hundredEvents.system27();
		hundredEvents.system28();
		hundredEvents.system29();
		hundredEvents.system30();
		hundredEvents.system31();
		hundredEvents.system32();
		hundredEvents.system33();
		hundredEvents.system34();
		hundredEvents.system35();
		
	}

}
